terms-conditions/participation-form.md

```diff
@@ -0,0 +1,94 @@
+### Certified FDC3 Conformance Program
```

Comment:
Should this be Certified FDC3 Agent conformance program?

(and then when we add apps, we have a separate section)

Reply:
yes, changing it